src/main/java/io/stargate/sgv2/jsonapi/service/processor/MeteredCommandProcessor.java

```diff
@@ -121,7 +121,12 @@ public <CommandT extends Command, SchemaT extends SchemaObject> Uni<CommandResul
             result -> {
               // --- Metrics Recording ---
               Tags tags = getCustomTags(commandContext, command, result);
-              sample.stop(meterRegistry.timer(jsonApiMetricsConfig.metricsName(), tags));
+              Timer timer = meterRegistry.timer(jsonApiMetricsConfig.metricsName(), tags);
+              sample.stop(timer);
+              String tenantId = commandContext.requestContext().getTenantId().orElse(UNKNOWN_VALUE);
+              commandContext
+                  .metricsTenantDeactivationConsumer()
+                  .trackMeterId(tenantId, timer.getId());
```

Comment:
Ok this is one place where metrics entries are produced (via `Timer`). But there are others too, like `CommandResolver` (line 223):

```
    meterRegistry.counter(jsonApiMetricsConfig.indexUsageCounterMetrics(), tags).increment();
```

and `TenantRequestMetricsFilter` (line 113)

```
      meterRegistry.counter(config.metricName(), tags).increment();
```

and likely others. Shouldn't they also call the same method?